torch/distributed/_composable/checkpoint.py

```diff
@@ -0,0 +1,127 @@
+import torch
```

Comment:
nit: wondering whether we can make the file name to be like 'checkpoint_activation.py"? so that it can be differentiated from model checkpoint in the future

Reply:
sure, will update.